documentation/src/main/asciidoc/public/reference/_backend-lucene.adoc

```diff
@@ -783,6 +783,40 @@ see the javadoc of `Similarity` and Lucene's source code.
 You can also find useful resources on the web, for example in Elasticsearch's documentation.
 ====
 
+[[backend-lucene-analysis-run-analysis]]
+=== Applying configured analyzers/normalizers to a string
+
+include::../components/_incubating-warning.adoc[]
+
+Hibernate Search provides an API that runs a configured analyzer/normalizer against a given string.
```

Comment:
It should work with built-in analyzers, too. Also I'm not sure one "runs" an analyzer?

```suggestion
Hibernate Search provides an API that applies an analyzer/normalizer to a given string.
```